src/backend/EdFi.DataManagementService.Backend.Postgresql/Deploy/Scripts/0001_Create_Documents_Table.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE IF NOT EXISTS documents (
+  id BIGINT GENERATED ALWAYS AS IDENTITY(START WITH 1 INCREMENT BY 1),
+  document_partition_key SMALLINT NOT NULL,
+  document_uuid CHAR(36) NOT NULL,
+  resource_name VARCHAR(256) NOT NULL,
```

Comment:
@bradbanister does MetaEd have a hard restriction on the length of a resource name?

Reply:
No. The limitation is the practicality of having a really long name in your MetaEd project and working with it in the editor.

Reply:
This could be problematic then, but I'm okay with leaving it as a reasonable upper limit.